stubs/pycocotools/pycocotools/mask.pyi

```diff
@@ -0,0 +1,31 @@
+from _typeshed import Incomplete
+from typing import Any, overload
+from typing_extensions import TypeAlias
+
+from . import _EncodedRLE
+
+# TODO: Use numpy types when #5768 is resolved.
+# import numpy as np
+# import numpy.typing as npt
+
+_NPUInt32: TypeAlias = Incomplete  # np.uint32
+_NDArrayUInt8: TypeAlias = Incomplete  # npt.NDArray[np.uint8]
+_NDArrayUInt32: TypeAlias = Incomplete  # npt.NDArray[np.uint32]
+_NDArrayFloat64: TypeAlias = Incomplete  # npt.NDArray[np.float64]
+
+def iou(
+    dt: _NDArrayUInt32 | list[float] | list[_EncodedRLE],
+    gt: _NDArrayUInt32 | list[float] | list[_EncodedRLE],
+    pyiscrowd: list[int] | _NDArrayUInt8,
+) -> list[Any] | _NDArrayFloat64: ...
```

Comment:
(No change requested, just pointing out to clarify:) This function _should_ have the parameters annotated as `list`, since the code explicitly forbids other kinds of sequences: https://github.com/ppwwyyxx/cocoapi/blob/71e284ef862300e4319aacd523a64c7f24750178/PythonAPI/pycocotools/_mask.pyx#L182